planet4-gutenberg-blocks.php

```diff
@@ -251,35 +255,12 @@ function set_allowed_block_types( $allowed_block_types, $post ) {
 		'campaign' => $campaign_block_types,
 	];
 
-	$allowed_p4_block_types = $all_allowed_p4_block_types[ $post->post_type ];
-
-	if ( empty( $allowed_p4_block_types ) ) {
-		return $wordpress_blocks;
-	}
-
-	$allowed_block_types = array_merge( $wordpress_blocks, $allowed_p4_block_types );
+	$allowed_p4_block_types = $all_allowed_p4_block_types[ $post_type ] ?? $all_allowed_p4_block_types[ 'page' ];
```

Comment:
Making this fall back to page block types fixes the issue of p4 blocks not being allow in the reusable block editor.

However there's also a chance this results in the block types being allow unintentionally on custom post types created by NROs. If this could be a problem I can preserve the exact previous behavior of only allowing core blocks.